core-war/src/main/webapp/util/javaScript/vuejs/components/address/silverpeas-address-search-input.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2000 - 2023 Silverpeas
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * As a special exception to the terms and conditions of version 3.0 of
+ * the GPL, you may redistribute this Program in connection with Free/Libre
+ * Open Source Software ("FLOSS") applications as described in Silverpeas's
+ * FLOSS exception.  You should have received a copy of the text describing
+ * the FLOSS exception, and it is also available here:
+ * "https://www.silverpeas.org/legal/floss_exception.html"
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+(function() {
+
+  const commonAsyncComponentRepository = new VueJsAsyncComponentTemplateRepository(
+      webContext + '/util/javaScript/vuejs/components/address/silverpeas-address-search-input-templates.jsp');
+
```

Comment:
I don't really understand what it is confusing in this naming.

Reply:
Because, by convention, the template and the js that motorizes it should have the same name (minus the template suffix for the template JSP). So any dev by the name will know the js linked to the template.

Reply:
Heu no. The JSP here contains only one single template. The one the JS motorizes.

Reply:
Ho, yes. I understand now what you were thinking for.
You are right, it exists only one template.
During the development, I wrote several components. But I kept only one at the end.
I will rename the file.